SDLMTCloud.Provider/Sdl.Community.MTCloud.Provider/Tests/UnitTests/Sdl.Community.MTCloud.Provider.UnitTests/RateItViewModelTests.cs

```diff
@@ -15,7 +16,7 @@ public RateItViewModelTests()
 			var providerAction = Substitute.For<IActionProvider>();
 			var segmentSupervisor = Substitute.For<ISegmentSupervisor>();
 			var messageBoxService = Substitute.For<IMessageBoxService>();
-			_rateItViewModel = new RateItViewModel(shortcutService, providerAction, segmentSupervisor, messageBoxService);
+			//_rateItViewModel = new RateItViewModel(shortcutService, providerAction, segmentSupervisor, messageBoxService, editorController);
```

Comment:
please remove commented code

Reply:
I just commented that line so I can return to it once I'm finished with the Rate It enhancement. There are a lot of commented lines in the tests, in fact, all the tests are commented right now because of the massive restructuring of the code. These will all need to be recalibrated to be able to use them.